easypost/services/shipment_service.py

```diff
@@ -62,12 +62,9 @@ def get_next_page(
         params = {
             "before_id": shipments["shipments"][-1].id,
             "page_size": page_size,
-            "include_children": shipments.get(_FILTERS_KEY, {}).get(
-                "include_children", None
-            ),  # Use the same include_children as the last page
-            "purchased": shipments.get(_FILTERS_KEY, {}).get(
-                "purchased", None
-            ),  # Use the same purchased as the last page
+            "include_children": shipments.get(_FILTERS_KEY, {}).get("include_children"),
+            # Use the same include_children as the last page
```

Comment:
this comment should be above what it's referencing if it's unable to be on the same line